packages/workbox-background-sync/demo/sw.js

```diff
@@ -1,35 +1,36 @@
 /* global workbox*/
 importScripts(
-	'../build/background-sync-queue.js',
-	'../../workbox-routing/build/workbox-routing.js',
-	'../../workbox-runtime-caching/build/workbox-runtime-caching.js'
+  '../build/importScripts/workbox-background-sync.dev.v0.0.1.js',
+  '../../workbox-routing/build/importScripts/workbox-routing.dev.v0.0.1.js',
+  '../../workbox-runtime-caching/build/importScripts/'
+    + 'workbox-runtime-caching.dev.v0.0.1.js'
 );
 
 // Have the service worker take control as soon as possible.
 self.addEventListener('install', (event) => {
-	event.waitUntil(self.skipWaiting());
+  event.waitUntil(self.skipWaiting());
 });
 self.addEventListener('activate', (event) => {
-	event.waitUntil(self.clients.claim());
+  event.waitUntil(self.clients.claim());
 });
 
 
 let bgQueue = new workbox.backgroundSync.QueuePlugin({callbacks:
-	{
-		onResponse: async(hash, res) => {
-			self.registration.showNotification('Background sync demo', {
-				body: 'Product has been purchased.',
-				icon: 'https://shop.polymer-project.org/images/shop-icon-384.png',
-			});
-		},
-		onRetryFailure: (hash) => {},
-	},
-	dbName: 'queues',
+  {
+    onResponse: async(hash, res) => {
```

Comment:
What's `hash` and `res` used for? Could you use them in the sample code, or else add a comment explaining what it's for?